testing/validation.ts

```diff
@@ -219,12 +219,12 @@ function tryParseDateTimeString(result: unknown, schema: BaseStringSchema) {
 
 function tryParseUrl(result: unknown, schema: BaseStringSchema) {
   const invalidUrlError = {
-    message: `Property with codaType "${schema.codaType}" must be a valid HTTP(S) url, but got "${result}".`,
+    message: `Property with codaType "${schema.codaType}" must be a valid HTTP(S) or data url, but got "${result}".`,
   };
   try {
     const url = urlParse(result as string);
 
-    if (!(url.protocol === 'http:' || url.protocol === 'https:')) {
+    if (!(url.protocol === 'http:' || url.protocol === 'https:' || url.protocol === 'data:')) {
```

Comment:
Totally unrelated, but while you're here...we dropped support for non-https urls a little while back, we might as well flag them too here. Mind removing the 'http:' check and updating the error message?

Reply:
Yep, done!